Git Testing/Scenes/Details/View/PictureDetailVC.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PictureDetailViewController.swift
+//  Git Testing
+//
+//  Created by BrainX Technologies on 14/01/2021.
+//
+
+import UIKit
+
+class PictureDetailVC: UIViewController {
+
+    @IBOutlet weak var theImageView: UIImageView!
```

Comment:
Always put a pragma mark above the related methods, properties and outlets so you can find them easily.